src/main/java/io/phdata/streamliner/schemadefiner/configbuilder/ScriptCommand.java

```diff
@@ -126,6 +126,8 @@ private static void build(
                           log.info("Rendering file: {} ", templateFile);
                           Map<String, Object> map = new LinkedHashMap<>();
                           map.put("configuration", configuration);
+                          map.put("configurationDiff", new ConfigurationDiff());
+                          map.put("tableDiff", new TableDiff());
```

Comment:
During initial script generation, configurationDIff and tableDiff won't be available. Hence providing empty object because templates does not accept null values